### PR TITLE
Fix broken android circle

### PR DESCRIPTION
**Pull request by JonghunYu**

### CODE REVIEW

**Comment by JonghunYu** [comment]
@vinayr I made a PR please review and publish it

File: example/package.json [hunk 1]
```diff
@@ -3,11 +3,21 @@
...
+    "start": "node node_modules/react-native/local-cli/cli.js start",
+    "test": "jest"
   },
   "dependencies": {
-    "react": "15.2.1",
-    "react-native": "0.30.0",
-    "react-native-simple-gauge": "latest"
+    "react": "16.0.0-alpha.12",
+    "react-native": "0.46.1",
+    "react-native-simple-gauge": "file:///Users/yuzong/work/react-native-simple-gauge"
```
Comment (vinayr) [inline example/package.json:12]: @JonghunYu change this

**Comment by vinayr** [comment]
0.1.3 published https://www.npmjs.com/package/react-native-simple-gauge

**Comment by JonghunYu** [comment]
@vinayr thanks let's have dinner this week

**Pull request merged**